production/helm/loki/README.md.gotmpl

```diff
@@ -16,140 +16,74 @@ Add the following repo to use the chart:
 helm repo add grafana https://grafana.github.io/helm-charts
 ```
 
-{{ template "chart.valuesSection" . }}
-
-## Components
-
-The chart supports the components shown in the following table.
-Ingester, distributor, querier, and query-frontend are always installed.
-The other components are optional.
-
-| Component | Optional | Enabled by default |
-| --- | --- | --- |
-| gateway |  ✅ |  ✅ |
-| write |  ❎ | n/a |
-| read |  ❎ | n/a |
+## Upgrading from v2.x
```

Comment:
Added upgrade/migration documentation. Currently only the first approach is spelled out as that's the only one tested in this PR.